### DISCUSSION TITLE
one class to make all shrink

### DISCUSSION THREAD

**Discussion by wnunezc** [twbs/bootstrap#31147] [General] [answered]
Hi guys I love BT, I and other developers who use BT for web pages I think we have all the best that the community has given behind this design framework.

But I've been wondering for a couple of years now; we can better optimize the space for Web Administrative environments; many clients and developers ask me about this from time to time.

the main problem is that BT is minimalist; while some administrative environments require a greater flow of visual data and the space used by BT for the Margins / Pading of each element is sometimes excessive but consistent.

I am looking for a way to bring BT 4 ~ 5 to that point with the use of only one class that is in charge of Decreasing / Eliminating all this space abuse. Something like:

`<body class = "bt-shrink">`

I think of the end user with a 1366 x 768 px screen with a datable in front.

It is not easy, I would like to know if there are other design frameworks that are already oriented towards this objective, or if BT plans to do a fork, such as BT for back-office business environments.

**Promise convert this threat to Feature Request if might acceptance.**

**Comment by mdo** [comment, marked as answer]
Not sure how this would work, but if I'm understanding you right, it's not something we'd want to tackle anyway. Thanks for the suggestion though!

**Comment by ffoodd** [comment]
@arcanisgk I have the same request several times, FYI one of the best to go through a condensed UI is to compile your own build with decreased values for spacings and font-sizes, mostly—or to use the small variants for each available components (eg. tables, for back-offices).

**Reply by wnunezc** [reply]
I've been looking for a tutorial all day to do it.
